lab_6_pipeline/pipeline.py

```diff
@@ -2,13 +2,32 @@
 Pipeline for CONLL-U formatting
 """
 from pathlib import Path
+import pymorphy2
```

Comment:
I have taken a look into your `1_full_conllu.conllu` - it has one sentence instead of many sentences like you have in `1_morphological_conllu.conllu`

[processed-dataset.zip](https://github.com/fipl-hse/2022-2-level-ctlr/files/11481427/processed-dataset.zip)